docs/kmip.md

```diff
@@ -118,6 +118,12 @@ The master key state polling functionality is particularly useful in cluster dep
 | **Type**            | int|
 | **Description**     | The period in seconds to check the state of the master encryption key. Default: 900. If the master encryption key is not in the Active state, the node logs the error and shuts down. Available starting with version 6.0.17-14.|
 
+| Configuration file  | {{optionlink('security.kmip.useLegacyProtocol')}}|
+|-------------------- | --------------------|
+| **Command line**    | `kmipUseLegacyProtocol`|
+| **Type**            | boolean|
+| **Description**     | When `true`, sets the KMIP protocol version 1.0 or 1.1. This option has no effect, because Percona Server for MongoDB uses KMIP version 1.0 by default. It exists for compatibility with MongoDB Enterprise Edition to to align configuration files and simplify the migration process to Percona Server for MongoDB. Available starting with version 6.0.21-18. |
```

Comment:
```suggestion
| **Description**     | When `true`, sets the KMIP protocol version 1.0 or 1.1. This option has no effect, because Percona Server for MongoDB uses KMIP version 1.0 by default. It exists for compatibility with MongoDB Enterprise Advanced to align configuration files and simplify the migration process to Percona Server for MongoDB. Available starting with version 6.0.21-18. |
```